pkg/anonymiser/anonymiser.go

```diff
@@ -66,26 +66,27 @@ func (a *anonymiser) ReadTable(tableName string, rowChan chan<- database.Row, op
 					continue
 				}
 
-				for name, faker := range Functions {
-					if fakerType != name {
-						continue
-					}
+				faker, found := Functions[fakerType]
+				if !found {
+					logger.WithField("anonymiser", fakerType).Error("Anonymiser is not found")
+					row[column] = fmt.Sprintf("Invalid anonymiser: %s", fakerType)
```

Comment:
Should be not just fail? This error is very fixable by a user and probably should be in any case.

Reply:
I do not see a way for anonymiser to stop the whole process, so I went with a simple and cheap solution. Actualy there is a bug with race condition as well as it always finishes with `panic: send on closed channel`, but I'm going to tacklethis in the next PR.

Reply:
Maybe add a TODO here so we can think of handling this properly?

Reply:
Good point, added